### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by NoelLincoln**
Hello 👋

I built this capstone project based on an air pollution API(https://openweathermap.org/api/air-pollution).
I was able to achieve the following 

-When the page loads, the webapp shows the list of states/cities that are filtered out by some the name.
- Along with the state/city name, I displayed  a numeric value
- When a user clicks (or taps) on a category item, the application navigates to the details page
- In the details page, the webapp retrieves data from the API to show detailed data in the category states.
- When a user clicks on the "Back" button (<), the user navigates to the home page.
- Wrote unit tests for pure functions in the Redux code.
- Wrote integration tests for the application using the React Testing Library.
- Mocked the access to the API, so that the tests don't send actual requests.
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should is stored in the Redux store.

- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- The project is deployed and accessible online using render [live-demo](https://air-pollution-stats.onrender.com/)
- Added a link to the online version of your app to the README file.

### CODE REVIEW

File: src/components/StateDetails.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/* eslint-disable react/jsx-one-expression-per-line */
```
Comment (elisha2kyakpo1) [inline src/components/StateDetails.js]: - [ ] Kindly enable linters and fix the error, linters helps to highlight common errors in our code in order to have a standard codebase.

File: src/components/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+  return (
+    <section className="home-container">
+      <div className="country">
+        <div className="country-img">
+          <div className="world-img">
+            <img src={worldimg} alt="" />
+          </div>
+          <div className="country-details">
+            <h1>{country}</h1>
```
Comment (elisha2kyakpo1) [inline src/components/HomePage.js:113]: - [ ] Let's make sure the our design follow the given design with color, typography, spaces, etc. I opened the app on my mobile phone and the design was different. Kindly open the deployed app on your mobile phone to see how the boxes are displayed and make changes.

File: src/components/StateDetails.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+              <p>Nitrogen monoxide:</p>
+              <p>{airQualityData.list[0].components.no}</p>
+            </li>
+            <li className="air-components-stat">
+              <p>Nitrogen dioxide:</p>
+              <p>{airQualityData.list[0].components.no2}</p>
+            </li>
+            <li className="air-components-stat">
+              <p> Ozone:</p>
+              <p>{airQualityData.list[0].components.o3}</p>
```
Comment (elisha2kyakpo1) [inline src/components/StateDetails.js:53]: - [ ] The details page design doesn't match the given design. Kindly follow the design given and make changes to the elements.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @NoelLincoln, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
👍 Display countries and cities.
👏 Good search functionality.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly fix the broken license link in the readme file. All the links should be working properly for a professional readme file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @NoelLincoln👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Implemented all changes requested by the previous reviewer✔️
- The app works as expected and Good design✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**